### PR TITLE
Feature/21 filter finish

### PR DESCRIPTION
**Pull request by timohuber**
- Order filtered contacts according to mailing
- Hardcoded filter includes firstname, lastname and contact language

### CODE REVIEW

File: pool/database/seed/seed_user.ml [hunk 1]
```diff
@@ -15,6 +15,38 @@ type person =
...
 type persons = person list [@@deriving show, yojson]
 
+let known_users =
+  let create_contact i =
+    let open Pool_common in
+    let open Pool_user in
+    let get = CCResult.get_exn in
+    let id = Id.create () in
+    let first_name =
+      "firstname" ^ CCInt.to_string i |> Firstname.create |> get
```
Comment (mabiede) [inline pool/database/seed/seed_user.ml:25]: use `Format.asprintf` instead of `^`

File: pool/database/seed/seed_user.ml [hunk 2]
```diff
@@ -15,6 +15,38 @@ type person =
 
 type persons = person list [@@deriving show, yojson]
 
+let known_users =
+  let create_contact i =
```
Comment (mabiede) [inline pool/database/seed/seed_user.ml:19]: why is this needed? We do have a random user generator.

File: pool/web/view/component/component_filter.ml [hunk 3]
```diff
@@ -191,6 +191,24 @@ let value_input language input_type ?value () =
          options
          selected
          ()
+     | Key.Languages languages ->
+       let selected =
+         single_value
+         >>= function[@warning "-4"]
+         | Language lang ->
+           CCList.find_opt
+             (fun language -> Pool_common.Language.equal language lang)
```
Comment (mabiede) [inline pool/web/view/component/component_filter.ml:200]:
```suggestion
             (Pool_common.Language.equal lang)
```

**Review by mabiede — CHANGES_REQUESTED** [review]

File: pool/database/seed/seed_user.ml (hunk 2 above)
Comment (timohuber) [reply pool/database/seed/seed_user.ml:19]: Yes, but as the users are randomly generated, tests can randomly fail (for example if the matcher test does not find any contacts for an experiment).

Comment (timohuber) [reply pool/database/seed/seed_user.ml:19]: That is possible, but it would make the tests depending on each other. So if we would change the order for example, it would break the test cases. What about adding an additional seed to the `setup_test` function?
